changelog.txt

```diff
@@ -8,6 +8,91 @@ Legend
++ AIFC: additional interface validation checks (#5267)
++ UI: Center Popups (#5274)
++ Forms: Minimum text field length (#5287)
++ Visualize difference for EOF (#5404)
++ README: Support Ukraine (#5407)
++ Add support for folder logic "mixed" (#5413)
++ Check if package name is allowed (#5416)
++ Add transport for deletions in generic serializer (#5434)
++ TABU: Add "jump" to SE16 (#5459)
+* TOBJ: Fix xslt serialization dump (#5247)
+* Save additional repo view settings (#5257)
+* ENHO,SFPF: Use correct transport (#5258)
+* Repo View: 'Order by path' considers object name (#5261)
+* Adjust diff algorithm (#5251)
+* Correct task name parallel processing (#5269)
+* Fix lost remote on favorite refresh (#5277)
+* Fix status for new sub-packages (#5278)
+* Avoid double deserialize of text symbols (#5273)
+* Set window title for log viewer (#5300)
+* WAPA: Fix repeated pull of BSP page (Fiori) (#5310)
+* DOMA: Fix serialize of translations (#5304)
+* TABL: Properly show inactive objects (#5303)
+* Add refresh after pull errors (#5312)
+* SICF: clear ICFALIASNO (#5338)
+* Persist "show folder view" option (#5343)
+* SRVB: Fix for Service Binding  (#5344)
+* Improve progress bar and log for pull (#5311)
+* FUGR: Sort includes (#5352)
+* DTEL: REF TO DATA for DDIC element support (#5350)
+* Jump: Switch execution order - first specific then generic (#5357)
+* Don't fail if a dictionary object is activated with warnings (#5362)
+* SOTS: Fix dump in case of lower case language code (#5381)
+* Fix DDIC errors related to ref to class/interface (#5369)
+* TTYP: Fix pull/activate table type ref to data type (#5384)
+* CLAS/INTF: Fix serialize/deserialize of documentation (#5374)
+* WEBI: Sort tables (#5390)
+* SRVD: Update zcl_abapgit_object_srvd.clas.abap (#5395)
+* SRVB: Fix uncaught exception (#5398)
+* Repo List: Fix sort (#5397)
+* Fix deserialize order (#5399)
+* ajson, Automatic Update (#5412)
+* DDLX: Delete before DDLS (#5414)
+* UENO: Fix diff related to release (#5422)
+* Record objects in transports (#5419)
+* ENHC,ENHS,ENSC,SPRX,TRAN: Fix "dynpro_send_in_background" (#5430)
+* Fix deserialize textpool for some objects (#5417)
+* DDLS: Avoid dump and show error (#5437)
+* CUS1: Fix jump (#5447)
+* CHDO: Avoid double pull and fix uninstall (#5439)
+* UI: Prefill transport popup (#5438)
+* Record objects in transports (#5435)
+* UI: Word-wrap long filenames in repo/stage views (#5424)
+* DOMA: Deletion of domains with append (#5445)
+* DEVC: Raise error for main/structure packages (#5444)
+* Improve LXE error handling (#5440)
+* UI: Fix escaping of textarea input (#5456)
+* Improve JSON syntax highlighter (#5450)
+* DEVC: Fix DYNPRO_SEND_IN_BACKGROUND (#5461)
+* Avoid logon when comparing with remote (#5458)
+* DDLS: Clear ABAP language version (#5474)
+* DDLX: Clear ABAP language version (#5476)
+* FUGR: Fix diff of includes (#5489)
+* DTDC: Support ABAP 2208 (#5478)
+* Create empty classes/interfaces before DDIC (#5393)
```

Comment:
this could be a breaking change?

```suggestion
! Create empty classes/interfaces before DDIC (#5393)
```

Reply:
didn't break anything in CI but might somewhere else